docs/HowToUpgrade.md

```diff
@@ -5,15 +5,15 @@
 A rolling upgrade updates a NuoDB domain, which may include multiple databases, to a new NuoDB release without shutting down the database(s).
 Rolling upgrades ensure continuous database availability throughout the upgrade process.
 
-A NuoDB domain deployed in Kubernetes using the NuoDB Helm Charts contains multiple Helm releases.
+A NuoDB domain deployed in Kubernetes using the NuoDB Helm charts contains multiple Helm releases.
```

Comment:
I think this sentence reads easier with the commas here, "... , using the NuoDB Helm charts, ..."